snuba/datasets/entities/metrics.py

```diff
@@ -48,8 +48,10 @@ def transform_expression(exp: Expression) -> Expression:
 
             key = exp.key
             if not isinstance(key.value, str) or not key.value.isdigit():
-                raise InvalidExpressionException(
-                    exp, "Expected a string key containing an integer in subscriptable."
+                raise InvalidExpressionException.from_args(
+                    exp,
+                    "Expected a string key containing an integer in subscriptable.",
+                    should_report=True,
```

Comment:
Why did you add `should_report=True`? That's the default already. Is this going to cause the exceptions to be reported when they weren't before?

Reply:
good point, not needed